cli/src/main/java/org/apache/uniffle/client/RestClientImpl.java

```diff
@@ -45,6 +50,7 @@ public class RestClientImpl implements RestClient {
   private static final Logger LOG = LoggerFactory.getLogger(RestClientImpl.class);
   private CloseableHttpClient httpclient;
   private String baseUrl;
+  final ObjectMapper mapper = new ObjectMapper().setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
Should this need to be `private`?

Reply:
Thanks for reviewing the code, I will modify code.